react-components/select/step3.md

```diff
@@ -0,0 +1,35 @@
+Now let's begin customizing. There are many options available to customize your select component and its options.
+
+One such customization, for both typeahead and multiple typeahead variants, is custom filtering.
+
+By default, these variants match the entered input with the options' values in order. For example, given the "Alabama" option value, an "Ala" text input would match but a "bama" text input would not.
+
+Let's allow any input to be matched, regardless of ordering.
+
+1. Add a custom filtering method to the App class. The component expects a function that will return a list of options to display. In the code below, we match the text input regardless of ordering using a regular expression, and santize the text input to avoid extraneous characters. Using the same example above, this will mean that a text input of "bama" would still match for the option "Alabama".
```

Comment:
Separate the action of the step from the paragraph of text, maybe put the paragraph inside of a `Note:`